x-pack/plugins/security_solution/public/management/components/artifact_list_page/hooks/use_with_artifact_list_data.ts

```diff
@@ -165,7 +165,7 @@ export const useWithArtifactListData = (
   return useMemo(
     () => ({
       isPageInitializing,
-      doesDataExist: doesDataExist ?? false,
+      doesDataExist: doesDataExist ?? true,
```

Comment:
I don't know the rest of the context here, but this line on its own seemed peculiar. Is assuming default of `true` safe? can `doesDataExist` ever be `undefined` or something from some errors?

Reply:
@pzl sorry this line was just to get around an empty splash screen, not intended to be part of this pr at all, thanks for catching.